src/odb/src/db/dbTechLayer.h

```diff
@@ -135,11 +136,11 @@ class _dbTechLayer : public _dbObject
   dbTable<_dbTechLayerMinCutRule>* min_cuts_rules_tbl_;
 
   dbTable<_dbTechLayerAreaRule>* area_rules_tbl_;
-
   // User Code Begin Fields
 
   uint _pitch_x;
   uint _pitch_y;
+  uint _first_last_pitch;
```

Comment:
please update the following functions to handle this variable:
- _dbTechLayer::out
- _dbTechLayer::differences
- _dbTechLayer::operator==
- **dbIStream& operator>>(dbIStream& stream, _dbTechLayer& obj)**
- **dbOStream& operator<<(dbOStream& stream, const _dbTechLayer& obj)**